src/aspire/commands/preprocess.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+import click
+
+from aspire.estimation.noise import WhiteNoiseEstimator
+from aspire.source.relion import RelionSource
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.option('--data_folder', default=None,
+              help='Path to data folder')
+@click.option('--starfile_in', required=True,
+              help='Path to input starfile relative to data folder')
+@click.option('--starfile_out', required=True,
+              help='Path to output starfile relative to data folder')
+@click.option('--pixel_size', default=1, type=float,
+              help='Pixel size of images in starfile')
+@click.option('--max_rows', default=None, type=int,
+              help='Max. no. of image rows to read from starfile')
+@click.option('--flip_phase', default=True,
+              help='perform phase flip or not')
+@click.option('--max_resolution', default=16, type=int,
+              help='Resolution of downsampled images read from starfile')
+@click.option('--normalize_background', default=True,
+              help='Whether to normalize images to background noise')
+@click.option('--whiten_noise', default=True,
+              help='Whiten background noise')
+@click.option('--invert_contrast', default=True,
+              help='Invert the contrast of images so molecules are shown in white')
+@click.option('--batch_size', default=512,
+              help='Batch size to load images from .mrc files.')
+@click.option('--save_mode', default='single',
+              help='Option to save mrc file, if not single, saved to multiply files in batch size')
+@click.option('--overwrite', default=False,
+              help='Whether to overwrite mrc files if they already exist')
+def preprocess(data_folder, starfile_in, starfile_out, pixel_size, max_rows,
+               flip_phase, max_resolution, normalize_background, whiten_noise,
+               invert_contrast, batch_size, save_mode, overwrite):
+    """
+    Preprocess the raw images and output desired images for future analysis
+    """
+    # Create a source object for 2D images
+    logger.info('This script illustrates how to preprocess experimental CryoEM images')
```

Comment:
Illustrates?

Reply:
Changed to "This script preprocesses experimental CryoEM images" in new commit.

Reply:
But why is this here? If the user is running it, don't they know already what it's supposed to do?

Reply:
It tends to print out the script  information to screen or file for future reading. This statement was removed in commit 13b2c79.